migration/identity-migration/src/main/java/io/camunda/migration/identity/dto/ClientType.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.migration.identity.dto;
+
+import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
+
+public enum ClientType {
```

Comment:
compare to https://github.com/camunda-cloud/identity/blob/main/management-api/src/main/java/io/camunda/identity/record/ClientType.java I went for an explicit UNKNOWN type in case there is any upstream change.